rundeckapp/grails-app/jobs/rundeck/quartzjobs/ExecutionsCleanUp.groovy

```diff
@@ -184,21 +184,38 @@ class ExecutionsCleanUp implements InterruptableJob {
             logger.info("found ${totalFound} executions")
             if(totalToExclude >0) {
                 if (minimumExecutionToKeep > 0) {
+
                     int totalExecutions = this.totalAllExecutions(executionService, project)
                     int sub = totalExecutions - totalToExclude
+
                     logger.info("minimum executions to keep: ${minimumExecutionToKeep}")
                     logger.info("total exections of project ${project}: ${totalExecutions}")
                     logger.info("total to exclude: ${totalToExclude}")
 
-                    //minimumExecutionToKeep should be triggered if the number of execution to delete is lower than maximum delete size
-                    if (totalToExclude < maximumDeletionSize && sub < minimumExecutionToKeep) {
+                    boolean removeSubResult= false
+                    boolean dontRemove= false
```

Comment:
code is probably more readable if the boolean isn't negated, e.g. `remove=true` instead of `dontRemove=false`

Reply:
done